welcome/oke_about.adoc

```diff
@@ -263,7 +264,7 @@ The following table is a summary of the feature availability in {oke} and {produ
 | Vertical Pod Autoscaler | Included | Included | Vertical Pod Autoscaler
 | Cluster Monitoring (Prometheus) | Included | Included | Cluster Monitoring
 | Device Manager (for example, GPU) | Included | Included | N/A
-| Log Forwarding (with fluentd) | Included | Included | Red Hat OpenShift Logging Operator (for log forwarding with fluentd)
+| Log Forwarding (with fluentd) | Not Included - Requires separate subscription | Included | Red Hat OpenShift Logging Operator (for log forwarding with fluentd)
```

Comment:
See [Table 2. Features in OpenShift Kubernetes Engine and OpenShift Container Platform](https://66035--docspreview.netlify.app/openshift-enterprise/latest/welcome/oke_about#feature-summary)

Reply:
why do we qualify "with fluentd" .  I'm not certain why that is (or ever was) relevant

Reply:
No idea, how can Red Hat limit the Forwarding using  subscription.

Reply:
AFAIK,  the annotations.valid-subscription in CSV is used to control the operator Visibility in OKE.   
```
oc get csv cluster-logging.v5.8.0 -o json |jq '.metadata.annotations."operators.openshift.io/valid-subscription"'
"[\"OpenShift Kubernetes Engine\", \"OpenShift Container Platform\", \"OpenShift Platform Plus\"]"
 oc get csv elasticsearch-operator.v5.8.0 -o json |jq '.metadata.annotations."operators.openshift.io/valid-subscription"'
"[\"OpenShift Container Platform\", \"OpenShift Platform Plus\"]"
oc get csv loki-operator.v5.8.0 -o json |jq '.metadata.annotations."operators.openshift.io/valid-subscription"'
"[\"OpenShift Container Platform\", \"OpenShift Platform Plus\"]"
```

Reply:
Sorry, I don't have answers for your questions, @jcantrill @anpingli. 
I think it would help to start from a zero state and answer the question: What do we need to tell customers about support for Logging in OKE and OCP?

Reply:
In other words, if a customer asked you, "Please can you tell me about support for Logging in OKE and OCP?", what would our reply be?

Reply:
@rolfedh I can't speak to the mechanics of the subscription or how it might be limited; I don't know if this is an honor thing (Jamie may Know) or if annotations restrict it.  With regards to my originally query,  I would drop reference of "with fluentd" or any reference to fluentd; I don't see the relevance

Reply:
@jcantrill What is Jamie's full name?

Reply:
@anpingli Thanks for sharing `annotations.valid-subscription`. 
If I'm reading it correctly, OKE includes `cluster-logging` but excludes `elasticsearch-operator` and `loki-operator`.
@jcantrill I'm not sure what Platform Logging or Log Forwarding consists of. Could you help me translate this information into statements about each one?